### PR TITLE
*: Merge pprof and status port and add flag to set host

### PR DESCRIPTION
**Pull request by shenli**
1. Merge pprof port with status port
2. Add flag to set host
3. Add flag to disable status report

@coocood @siddontang @BusyJay @tiancaiamao PTAL

### CODE REVIEW

**Comment by coocood** [comment]
LGTM

**Comment by zimulala** [comment]
LGTM

File: tidb.go [hunk 1]
```diff
@@ -289,10 +281,6 @@ func init() {
 	// Register default memory and goleveldb storage
 	RegisterLocalStore("memory", goleveldb.MemoryDriver{})
 	RegisterLocalStore("goleveldb", goleveldb.Driver{})
-	// start pprof handlers
-	if EnablePprof {
-		go http.ListenAndServe(PprofAddr, nil)
```
Comment (siddontang) [inline tidb.go]: how to see pprof now?

Comment (shenli) [inline tidb.go]: With the same port of prometheus which can the set by --status flag.

**Comment by siddontang** [comment]
LGTM

**Pull request merged**